engine/src/main/java/com/blobcity/db/sql/statements/SelectExecutor.java

```diff
@@ -361,9 +356,7 @@ private String produceResult(final String ds, final String collection, final Str
                 .put(BQueryParameters.PAYLOAD, result)
                 .put(BQueryParameters.TIME, executionTime)
                 .put(BQueryParameters.ROWS, result.size()).toString();
-        if(LicenseRules.QUERY_RESULT_CACHING) {
```

Comment:
Keep the `QUERY_RESULT_CACHING` check, as we would like this to be user controllable. The user should be able to dynamically enable and disable `QUERY_RESULT_CACHING`.

Reply:
Restored.